Sources/Commands/SwiftTestTool.swift

```diff
@@ -111,6 +111,10 @@ struct TestToolOptions: ParsableArguments {
             help: "Path where the xUnit xml file should be generated.")
     var xUnitOutput: AbsolutePath?
 
+    @Option(name: .customLong("json-output"),
+            help: "Path where the json output file should be generated.")
+    var jsonOutput: AbsolutePath?
```

Comment:
this flag should be mutually exclusive with with xUnitOutput?

Reply:
Good questions, I don't see a reason why they have to be mutually exclusive besides modifying the options from the command.

@MaxDesiatov I was thinking about this as well, but giving that I'm not familiar with the project I decided to keep this patch it as simple as possible. Would it be alright if we try to add the output option refactor in a following up patch?

Reply:
yes, I think it is fine for now